ui/src/timeMachine/components/ToolbarTab.tsx

```diff
@@ -4,20 +4,31 @@ import React, {PureComponent} from 'react'
 // Styles
 import 'src/timeMachine/components/ToolbarTab.scss'
 
-interface Props {
+interface PassedProps {
```

Comment:
_nit_ We've been using `StateProps`, `DispatchProps`, and `OwnProps` for `mstp`, `mdtp`, and passed props, respectively.

```suggestion
interface OwnProps {
```